syft/frameworks/torch/tensors/pointer.py

```diff
@@ -1,4 +1,5 @@
 from .abstract import AbstractTensor
+from syft.codes import MSGTYPE
```

Comment:
`from syft import codes` is preferable, but since the imports in other files do not follow this I suggestcreate a bug and fix it later.